lib/travis/build/script/rust.rb

```diff
@@ -18,7 +18,7 @@ def export
         def setup_cache
           if data.cache?(:cargo)
             sh.fold 'cache.cargo' do
-              directory_cache.add "${TRAVIS_HOME}/.cargo", "target"
+              directory_cache.add "${TRAVIS_HOME}/.cargo", "target", "${TRAVIS_HOME}/.rustup", "${TRAVIS_HOME}/.cache/sccache"
```

Comment:
Note to self: We probably want a separate constant or a method to refer to these directories.